src/simulator.zig

```diff
@@ -26,14 +26,15 @@ var cluster: *Cluster = undefined;
 pub fn main() !void {
     // TODO Use std.testing.allocator when all deinit() leaks are fixed.
     const allocator = std.heap.page_allocator;
+    const ga = std.testing.allocator;
 
     var args = std.process.args();
 
     // Skip argv[0] which is the name of this executable:
-    _ = args.nextPosix();
+    _ = args_next(&args, ga);
```

Comment:
Good idea to use `_` to ignore the return value instead of introducing `prog_name` as per the previous PR.